README.md

```diff
@@ -45,7 +45,9 @@ For example,
     key_id: https://testnotationakv.vault.azure.net/keys/notationLeafCert/c585b8ad8fc542b28e41e555d9b3a1fd
     target_artifact_reference: myRegistry.azurecr.io/myRepo@sha256:aaabbb
     signature_format: cose
-    plugin_config: ca_certs=.github/cert-bundle/cert-bundle.crt
+    plugin_config: |-
+      ca_certs=.github/cert-bundle/cert-bundle.crt
+      self_signed=false
```

Comment:
I am wondering what if the user accidentally passes a YAML object to `plugin_config` instead of a string? Which component will report error? What does the error look like?

Reply:
Updated to JSON array string as input.